discovery-infra/tests/base_test.py

```diff
@@ -0,0 +1,148 @@
+import os
+import pytest
+
+import assisted_service_api
+import utils
+import consts
+from tests.conftest import EnvVariable
+
+if os.environ.get('NODE_ENV') == 'QE_VM':
+    from node_controlers.qe_vm_controler import QeVmController as node_controler
+else:
+    raise NotImplementedError
+
+class BaseTest:
+    @pytest.fixture()
+    def api_client(self, request):
```

Comment:
why not to add api client to init?
much easier will be to use it. Not sure we we need it as fixture. same with controller

Reply:
in test u will just use it as self.api_client.do_smth()

Reply:
and then all the functions here will not need to get it from outside, i think it is much more easier

Reply:
pytest classes cannot have an __init__ or __del__ method other than the defaults

Reply:
hmm. right everything works with fixtures